en/best-layout-practices.md

```diff
@@ -19,13 +19,13 @@ This section provides design guidance for the vertical layout of Components and
 
 ### Vertical Centering
 
-In order for a layout to always be in the vertical center of the group or artboard it belongs to, its Components and Patterns have to be grouped together in a new group with fixed height, which is positioned at equal distances from the top and bottom borders of the original group or artboard mentioned above. Also make sure that the new group is neither top, nor bottom pinned.
+In order for a layout always to be in the vertical center of the group or artboard it belongs to, it is Components and Patterns have to be grouped together in a new group with fixed height, which is positioned at equal distances from the top and bottom borders of the original group or artboard mentioned above. In addition, make sure that the new group is neither top, nor bottom pinned.
```

Comment:
it is Components and Patterns -> its Components and Patterns 
Possessive form of the noun is intended here. Meaning the components/patterns that belong to the artboard.